docs/interactions.rst

```diff
@@ -0,0 +1,123 @@
+:orphan:
+
+.. currentmodule:: nextcord
+
+.. _client_index:
+   
+Interaction Commands
+======================
+
+As discord have added interaction commands a feature with alot of possiblities and ideas we at nextcord have decided to add them to our fleet of features,
+
+This doc will explain the innerworkings and how to use interaction commmands.
+
+We suggest you learn how to make regular commands before looking through here.
+
+How To Make A Simple Interaction Command
+------------------------------------------
+
+This right here is a simple ping pong Command made with Nextcords slash feature.
+
+.. code-block:: python3
+
+    @bot.slash_command(name="ping")
+    async def ping(interaction):
+        await interaction.response.send_message("Pong!")
+        
+The way it works is that you use the slash_command function to interact with the DiscordAPI "name" is the name of your slash command, And guild_ids is used to limit the guilds that the slash command is available also useful for testing as slash commands can take up to an hour to register
+
+How To Use Sub-Commands
+-------------------------
+
+The way sub-commands work is that you will make a normal slash command that will never be called then make the sub-commands and have them do the work of real slash commands, There is no difference to slash commands and sub-commands, The Only thing you will need to change is functions. 
```

Comment:
Again I am suggesting small changes to grammar and punctuation.  It is in my opinion however this might benefit from a rewrite of the technical aspect. 

"The way sub-commands work is that you make a normal slash command that will never be called, and then make the sub-commands and have them do the work of real slash commands. There is no difference with slash commands and sub-commands. The only thing you will need to change is functions."